ui/app/scripts/controllers/serviceVersionStatusCtrl.js

```diff
@@ -50,6 +50,11 @@ angular.module('apollo')
                     }
                     apolloApiService.getGroupsPerServiceAndEnvironment($scope.selectedEnvironment.id, $scope.selectedService.id).then(function (response) {
                         $scope.serviceGroups = response.data;
+                        $scope.serviceGroups.sort(function(a, b){
```

Comment:
The best practice is that the data will arrive already sorted from ```getGroupsPerServiceAndEnvironment```, why you chose to sort it here?